app/jobs/process_delius_data_job.rb

```diff
@@ -54,14 +54,20 @@ def map_delius_to_case_info(delius_record)
         com_name: delius_record.offender_manager,
         crn: delius_record.crn,
         tier: map_tier(delius_record.tier),
+        local_delivery_unit: map_ldu(delius_record.ldu_code),
         team: team,
+        team_name: team&.name,
         case_allocation: delius_record.service_provider,
         welsh_offender: map_welsh_offender(delius_record.ldu_code),
         mappa_level: map_mappa_level(delius_record.mappa_levels)
       )
     end
   end
 
+  def map_ldu(ldu_code)
+    LocalDeliveryUnit.find_by(enabled: true, code: ldu_code)
```

Comment:
It might be nicer to drop the `enabled: true` condition here, and instead implement it in the `OffenderBase.ldu` private method.

That way, the new LDU mappings will still be captured, and can be 'switched on' immediately by flipping the `enabled` switch.

Otherwise, with the current implementation, we'd need to enable the LDU and then wait overnight for this process job to run and add the new LDU mapping.